renovate.json

```diff
@@ -1,5 +1,5 @@
 {
-  "pinVersions": true,
+  "extends": [":pinAllExceptPeerDependencies"],
   "semanticCommits": true,
   "depTypes": [{ "depType": "dependencies", "pinVersions": false }],
```

Comment:
@rarkins Looking at the updated https://github.com/apollographql/apollo-server/pull/954, perhaps there's another configuration change necessary here too.

It seems that `depTypes` might be deprecated (I don't see it [on the configuration options](https://renovateapp.com/docs/configuration-reference/configuration-options#deptypelist)) or it's not getting respected here.  Perhaps I should get rid of `depTypes` (line 4 here) and use `"extends": [":pinOnlyDevDependencies"]` (rather than `:pinAllExceptPeerDependencies`) — since that seems to be what the configuration was (trying) to define before.

Thoughts?